src/context/Localization/language/en.json

```diff
@@ -555,7 +555,14 @@
     "ReviewTransaction": "Review the Transaction",
     "SignTransaction": "Sign Transaction",
     "DenyTransaction": "DenyTransaction",
-    "transactionToastMessage": "If the transaction isn’t visible, wait a moment and refresh."
+    "transactionToastMessage": "If the transaction isn’t visible, wait a moment and refresh.",
```

Comment:
Could we give it a more "successful" messgae? Ie. Transaction sent successfully! then the rest of the text